pkg/apis/navigator/v1alpha1/types.go

```diff
@@ -265,6 +268,7 @@ type ElasticsearchPilotStatus struct {
 	// an unknown number of documents, whereas 0 indicates that the node is
 	// empty
 	Documents *int64 `json:"documents,omitempty"`
+	Version   string `json:"version,omitempty"`
```

Comment:
Can this be a semver instance? For consistency with `ElasticsearchCluster.Version`.